docs/section19/19-9.md

```diff
@@ -0,0 +1,32 @@
+# Section 19 - Rule 19-9     
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-9     
+**Rule Description:** Air economizers shall not be included in baseline HVAC Systems 1, 2, 9, and 10.    
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.6      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline HVAC system types 1, 2, 9, and 10.
+
+**Function Calls:**  
+1. get_baseline_system_types  
+
+## Rule Logic:   
+**Applicability Check 1**   
+- Create ditionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMR)`  
+- Check if B-RMR is modeled with at least one air-side system that is Type-1,1a,1b,1c,2,9,9b,10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-1", "SYS-1a", "SYS-1b", "SYS-1c", "SYS-2", "SYS-9", "SYS-9b", "SYS-10"]):`
```

Comment:
We do not need this applicability check since the `baseline_ssytem_type_dict` will have the full list of baseline HVAC system type as keys.
See return data in the [`baseline_system_type_dict`](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_baseline_system_types.md?plain=1#L177)

Reply:
@weilixu I modified this check based on your comment.